pivy-importer/src/main/groovy/com/linkedin/python/importer/ivy/IvyFileWriter.groovy

```diff
@@ -16,49 +16,40 @@
 package com.linkedin.python.importer.ivy
 
 import com.linkedin.python.importer.pypi.VersionEntry
-import groovy.transform.TupleConstructor
+import com.linkedin.python.importer.deps.SdistDownloader
+import com.linkedin.python.importer.deps.WheelsDownloader
 import groovy.xml.MarkupBuilder
 import org.apache.commons.io.FilenameUtils
 
-@TupleConstructor
 class IvyFileWriter {
```

Comment:
Could you just keep the `@TupleConstructor` annotation here and then leave out the constructor below?